File: code_test/programmers/폰켓몬.py
def solution(nums):
    answer = 0
    n = len(nums)
    limit = n // 2
    list = []
    for i in nums:
        if i not in list:
            list.append(i)
    length = len(list)
    if (length < limit):
        answer = length
    else:
        answer = limit
    return answer

File: achieve/code_test/programmers/폰켓몬.js
function solution(nums) {
  var answer = 0;
  let list = [];
  nums.forEach((item) => {
    if (!list.includes(item)) {
      list.push(item);
    }
  });
  answer = list.length;
  let limit = nums.length / 2;
  answer = answer < limit ? answer : limit;
  return answer;
}
